napalm/__init__.py

```diff
@@ -26,6 +27,7 @@ def get_network_driver(vendor):
         'JUNOS': JunOSDriver,
         'JUNIPER': JunOSDriver,
         'FORTIOS': FortiOSDriver,
+#        'IBM': IBMDriver,
```

Comment:
As @fooelisa said, package it with pip and I will fix the requirements and the link. Don't worry about that. By the way, with minimum OS version I meant the network operating system that the device has to run.